src/Microsoft.ML/Runtime/EntryPoints/TrainTestMacro.cs

```diff
@@ -23,8 +24,14 @@ public sealed class SubGraphInput
 
         public sealed class SubGraphOutput
         {
-            [Argument(ArgumentType.Required, HelpText = "The model", SortOrder = 1)]
+            [Argument(ArgumentType.AtMostOnce, HelpText = "The predictor model", SortOrder = 1)]
             public Var<IPredictorModel> Model;
```

Comment:
>Model [](start = 40, length = 5)

Can you name it PredictorModel?
it's confusing if you start read code
in one place you have this:
            varName = input.Outputs.UseTransformModel ? input.Outputs.TransformModel.VarName : input.Outputs.Model.VarName;
in other place
string outputVarName = input.Outputs.UseTransformModel ? node.GetOutputVariableName(nameof(Output.TransformModel)) : 
                node.GetOutputVariableName(nameof(Output.PredictorModel));